guides/keras_hub/stable_diffusion_3_in_keras_hub.py

```diff
@@ -107,7 +107,7 @@ def display_generated_images(images):
 
 
 backbone = keras_hub.models.StableDiffusion3Backbone.from_preset(
-    "stable_diffusion_3_medium", height=512, width=512, dtype="float16"
+    "stable_diffusion_3_medium", image_shape=(512, 512, 3), dtype="float16"
```

Comment:
Why does the channel axis need to be specified? Other APIs that an image_shape arg only do height/width

Reply:
Is it true? I think most of the backbones in kerashub expect `(h, w, c)` format using `image_shape`.
https://github.com/search?q=repo%3Akeras-team%2Fkeras-hub%20image_shape&type=code

This change was requested by @divyashreepathihalli and I agree that it is more consistent with other backbone APIs

Additionally, even though most users won’t do this, it is still valid to train a diffusion model with the non-standard RGB images.

Reply:
Ok, sounds good